src/v/cluster/tests/health_monitor_test.cc

```diff
@@ -38,6 +38,8 @@
 
 static cluster::cluster_report_filter get_all{};
 
+static ss::logger logger{"health_monitor_test"};
```

Comment:
i think if you are inside a fixture test that you there is already a logger configured like `info("asdf {}", 333)`